target/linux/ath79/image/generic.mk

```diff
@@ -676,6 +676,15 @@ define Device/librerouter_librerouter-v1
 endef
 TARGET_DEVICES += librerouter_librerouter-v1
 
+define Device/mikrotik_rb-wapg-5hact2hnd
+  $(Device/rb-nor-flash-16M-ac)
+  ATH_SOC := qca9556
+  DEVICE_MODEL := RouterBOARD wAP G-5HacT2HnD (wAP AC)
+  DEVICE_PACKAGES += ath10k-firmware-qca988x-ct
+  SUPPORTED_DEVICES += rb-wapg-5hact2hnd mikrotik_rb-wapg-5hact2hnd
```

Comment:
The second one is not required. The one matching the compatible will be created automatically.

Reply:
My fault. Fixed.